### ISSUE TITLE
Swipe nav on Android is very laggy

### ISSUE THREAD

**Issue by nalinbhardwaj** [daimo-eth/daimo#562]
https://github.com/daimo-eth/daimo/assets/6984346/c01732c5-d505-4f0a-93a7-2ce88d9306b2

Pixel 3, in the last few seconds I am not making any taps/swipes

**Comment by nalinbhardwaj** [comment]
It seems this has something to do with my account being very large - the swipe laginess goes away on a clean testnet account. In theory we shouldn't ever need to render O(transactions) components for any accounts (and we don't for the obvious candidates - history list and recent recipients), so its unclear where this comes from.

Alternately, this may have to do with large memory consumption due to the account size being large -- confirm if so?

**Comment by nalinbhardwaj** [comment]
potentially related debug log entry:

```
2023-12-20T07:52:37.818Z log VirtualizedList: You have a large list that is slow to update - make sure your renderItem function renders components that follow React performance best practices like PureComponent, shouldComponentUpdate, etc. {"dt":8364,"prevDt":9384,"contentLength":6916.66650390625}
```

haven't seen that before

**Comment by dcposch** [comment]
fixed in #602
